src/main/java/net/dv8tion/jda/bot/sharding/ShardManager.java

```diff
@@ -104,6 +105,22 @@ default int getShardsTotal()
         return this.getShardsQueued() + this.getShardsRunning();
     }
 
+    /**
+     * The SelfUser of the logged in account.
+     *
+     * @throws java.lang.IllegalStateException
+     *         If there is no running shard
+     *
+     * @return The {@link net.dv8tion.jda.core.entities.SelfUser} for the currently logged in account.
+     */
+    default SelfUser getSelfUser()
+    {
+        return this.getShardCache().stream()
+                .findAny()
+                .orElseThrow(() -> new IllegalStateException("no active shards"))
+                .getSelfUser()
```

Comment:
I somehow don't believe this would compile

Reply:
Don't know how I missed that one, will fix.